RescuWise/settings.py

```diff
@@ -23,7 +23,7 @@
 SECRET_KEY = 'ja391197!tv&r$qbptd10(ecx0visy&ur#*)se&+*j(+v9#fbp'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```

Comment:
Make sure to always set debug back to False when you commit. Ideally we should set this up as an environment variable.

Reply:
@AshleyMatula can you fix this since this is your pr?